runtime error: invalid memory address or nil pointer dereference

**How to categorize this issue?**
<!--
Please select area, kind, and priority for this issue. This helps the community categorizing it.
Replace below TODOs or exchange the existing identifiers with those that fit best in your opinion.
If multiple identifiers make sense you can also state the commands multiple times, e.g.
  /area control-plane
  /area auto-scaling
  ...

"/area" identifiers:     audit-logging|auto-scaling|backup|certification|control-plane-migration|control-plane|cost|delivery|dev-productivity|disaster-recovery|documentation|high-availability|logging|metering|monitoring|networking|open-source|operations|ops-productivity|os|performance|quality|robustness|scalability|security|storage|testing|usability|user-management
"/kind" identifiers:     api-change|bug|cleanup|discussion|enhancement|epic|impediment|poc|post-mortem|question|regression|task|technical-debt|test
"/priority" identifiers: normal|critical|blocker
-->
/kind bug
/platform alicloud

**What happened**:

```
E0318 08:29:42.987561    6898 runtime.go:78] Observed a panic: "invalid memory address or nil pointer dereference" (runtime error: invalid memory address or nil pointer dereference)
goroutine 824 [running]:
k8s.io/apimachinery/pkg/util/runtime.logPanic(0x321e8e0, 0x55259f0)
	/go/src/github.com/gardener/gardener-extension-provider-alicloud/vendor/k8s.io/apimachinery/pkg/util/runtime/runtime.go:74 +0xa6
k8s.io/apimachinery/pkg/util/runtime.HandleCrash(0x0, 0x0, 0x0)
	/go/src/github.com/gardener/gardener-extension-provider-alicloud/vendor/k8s.io/apimachinery/pkg/util/runtime/runtime.go:48 +0x89
panic(0x321e8e0, 0x55259f0)
	/usr/local/go/src/runtime/panic.go:969 +0x1b9
github.com/gardener/gardener/pkg/controllerutils.tryPatchFinalizers.func1(0x10786bc, 0x552b1e0)
	/go/src/github.com/gardener/gardener-extension-provider-alicloud/vendor/github.com/gardener/gardener/pkg/controllerutils/finalizers.go:66 +0xea
k8s.io/client-go/util/retry.OnError.func1(0x0, 0x0, 0x0)
	/go/src/github.com/gardener/gardener-extension-provider-alicloud/vendor/k8s.io/client-go/util/retry/util.go:51 +0x3c
k8s.io/apimachinery/pkg/util/wait.runConditionWithCrashProtection(0xc0009e3790, 0x31fa000, 0x0, 0x0)
	/go/src/github.com/gardener/gardener-extension-provider-alicloud/vendor/k8s.io/apimachinery/pkg/util/wait/wait.go:211 +0x69
k8s.io/apimachinery/pkg/util/wait.ExponentialBackoff(0x989680, 0x4014000000000000, 0x3fb999999999999a, 0x4, 0x0, 0xc0009e3790, 0x100c9c8, 0x344b1e0)
	/go/src/github.com/gardener/gardener-extension-provider-alicloud/vendor/k8s.io/apimachinery/pkg/util/wait/wait.go:399 +0x55
k8s.io/client-go/util/retry.OnError(0x989680, 0x4014000000000000, 0x3fb999999999999a, 0x4, 0x0, 0x380b580, 0xc0009e3830, 0x32cfd00, 0x344b228)
	/go/src/github.com/gardener/gardener-extension-provider-alicloud/vendor/k8s.io/client-go/util/retry/util.go:50 +0xa6
k8s.io/client-go/util/retry.RetryOnConflict(...)
	/go/src/github.com/gardener/gardener-extension-provider-alicloud/vendor/k8s.io/client-go/util/retry/util.go:104
github.com/gardener/gardener/pkg/controllerutils.tryPatchFinalizers(0x39eca80, 0xc000da2c00, 0x0, 0x0, 0x2eb30ed8, 0xc0008273b0, 0x3a2ad40, 0xc00000d0e0, 0x380c800, 0x372ec10, ...)
	/go/src/github.com/gardener/gardener-extension-provider-alicloud/vendor/github.com/gardener/gardener/pkg/controllerutils/finalizers.go:60 +0x168
github.com/gardener/gardener/pkg/controllerutils.EnsureFinalizer(...)
	/go/src/github.com/gardener/gardener-extension-provider-alicloud/vendor/github.com/gardener/gardener/pkg/controllerutils/finalizers.go:49
github.com/gardener/gardener/extensions/pkg/controller/infrastructure.(*reconciler).reconcile(0xc000928910, 0x39eca80, 0xc000da2c00, 0x39f8560, 0xc000e10390, 0xc00000d0e0, 0xc00093b9e0, 0x36f92d7, 0x6, 0x0, ...)
	/go/src/github.com/gardener/gardener-extension-provider-alicloud/vendor/github.com/gardener/gardener/extensions/pkg/controller/infrastructure/reconciler.go:127 +0x11f
github.com/gardener/gardener/extensions/pkg/controller/infrastructure.(*reconciler).Reconcile(0xc000928910, 0x39eca80, 0xc000da2c00, 0xc000397b20, 0x1a, 0xc000f7d920, 0x9, 0x3a2ad40, 0xc00000cd20, 0x0, ...)
	/go/src/github.com/gardener/gardener-extension-provider-alicloud/vendor/github.com/gardener/gardener/extensions/pkg/controller/infrastructure/reconciler.go:121 +0x6c8
github.com/gardener/gardener/extensions/pkg/controller.(*operationAnnotationWrapper).Reconcile(0xc0009a6240, 0x39eca80, 0xc000da2c00, 0xc000397b20, 0x1a, 0xc000f7d920, 0x9, 0xc000da2c00, 0x100c29f, 0xc00003c000, ...)
	/go/src/github.com/gardener/gardener-extension-provider-alicloud/vendor/github.com/gardener/gardener/extensions/pkg/controller/reconciler.go:75 +0x259
sigs.k8s.io/controller-runtime/pkg/internal/controller.(*Controller).reconcileHandler(0xc0009977c0, 0x39ec9c0, 0xc000a3c440, 0x32cfde0, 0xc000d99cc0)
	/go/src/github.com/gardener/gardener-extension-provider-alicloud/vendor/sigs.k8s.io/controller-runtime/pkg/internal/controller/controller.go:263 +0x317
sigs.k8s.io/controller-runtime/pkg/internal/controller.(*Controller).processNextWorkItem(0xc0009977c0, 0x39ec9c0, 0xc000a3c440, 0x0)
	/go/src/github.com/gardener/gardener-extension-provider-alicloud/vendor/sigs.k8s.io/controller-runtime/pkg/internal/controller/controller.go:235 +0x205
sigs.k8s.io/controller-runtime/pkg/internal/controller.(*Controller).Start.func1.1(0x39ec9c0, 0xc000a3c440)
	/go/src/github.com/gardener/gardener-extension-provider-alicloud/vendor/sigs.k8s.io/controller-runtime/pkg/internal/controller/controller.go:198 +0x4a
k8s.io/apimachinery/pkg/util/wait.JitterUntilWithContext.func1()
	/go/src/github.com/gardener/gardener-extension-provider-alicloud/vendor/k8s.io/apimachinery/pkg/util/wait/wait.go:185 +0x37
k8s.io/apimachinery/pkg/util/wait.BackoffUntil.func1(0xc000bb6f50)
	/go/src/github.com/gardener/gardener-extension-provider-alicloud/vendor/k8s.io/apimachinery/pkg/util/wait/wait.go:155 +0x5f
k8s.io/apimachinery/pkg/util/wait.BackoffUntil(0xc0009e3f50, 0x398e960, 0xc000744060, 0xc000a3c401, 0xc000eb4cc0)
	/go/src/github.com/gardener/gardener-extension-provider-alicloud/vendor/k8s.io/apimachinery/pkg/util/wait/wait.go:156 +0xad
k8s.io/apimachinery/pkg/util/wait.JitterUntil(0xc000bb6f50, 0x3b9aca00, 0x0, 0x1, 0xc000eb4cc0)
	/go/src/github.com/gardener/gardener-extension-provider-alicloud/vendor/k8s.io/apimachinery/pkg/util/wait/wait.go:133 +0x98
k8s.io/apimachinery/pkg/util/wait.JitterUntilWithContext(0x39ec9c0, 0xc000a3c440, 0xc000a94bd0, 0x3b9aca00, 0x0, 0x1)
	/go/src/github.com/gardener/gardener-extension-provider-alicloud/vendor/k8s.io/apimachinery/pkg/util/wait/wait.go:185 +0xa6
k8s.io/apimachinery/pkg/util/wait.UntilWithContext(0x39ec9c0, 0xc000a3c440, 0xc000a94bd0, 0x3b9aca00)
	/go/src/github.com/gardener/gardener-extension-provider-alicloud/vendor/k8s.io/apimachinery/pkg/util/wait/wait.go:99 +0x57
created by sigs.k8s.io/controller-runtime/pkg/internal/controller.(*Controller).Start.func1
	/go/src/github.com/gardener/gardener-extension-provider-alicloud/vendor/sigs.k8s.io/controller-runtime/pkg/internal/controller/controller.go:195 +0x4e7
E0318 08:29:42.988966    6898 runtime.go:78] Observed a panic: "invalid memory address or nil pointer dereference" (runtime error: invalid memory address or nil pointer dereference)
goroutine 824 [running]:
k8s.io/apimachinery/pkg/util/runtime.logPanic(0x321e8e0, 0x55259f0)
	/go/src/github.com/gardener/gardener-extension-provider-alicloud/vendor/k8s.io/apimachinery/pkg/util/runtime/runtime.go:74 +0xa6
k8s.io/apimachinery/pkg/util/runtime.HandleCrash(0x0, 0x0, 0x0)
	/go/src/github.com/gardener/gardener-extension-provider-alicloud/vendor/k8s.io/apimachinery/pkg/util/runtime/runtime.go:48 +0x89
panic(0x321e8e0, 0x55259f0)
	/usr/local/go/src/runtime/panic.go:969 +0x1b9
k8s.io/apimachinery/pkg/util/runtime.HandleCrash(0x0, 0x0, 0x0)
	/go/src/github.com/gardener/gardener-extension-provider-alicloud/vendor/k8s.io/apimachinery/pkg/util/runtime/runtime.go:55 +0x10c
panic(0x321e8e0, 0x55259f0)
	/usr/local/go/src/runtime/panic.go:969 +0x1b9
github.com/gardener/gardener/pkg/controllerutils.tryPatchFinalizers.func1(0x10786bc, 0x552b1e0)
	/go/src/github.com/gardener/gardener-extension-provider-alicloud/vendor/github.com/gardener/gardener/pkg/controllerutils/finalizers.go:66 +0xea
k8s.io/client-go/util/retry.OnError.func1(0x0, 0x0, 0x0)
	/go/src/github.com/gardener/gardener-extension-provider-alicloud/vendor/k8s.io/client-go/util/retry/util.go:51 +0x3c
k8s.io/apimachinery/pkg/util/wait.runConditionWithCrashProtection(0xc0009e3790, 0x31fa000, 0x0, 0x0)
	/go/src/github.com/gardener/gardener-extension-provider-alicloud/vendor/k8s.io/apimachinery/pkg/util/wait/wait.go:211 +0x69
k8s.io/apimachinery/pkg/util/wait.ExponentialBackoff(0x989680, 0x4014000000000000, 0x3fb999999999999a, 0x4, 0x0, 0xc0009e3790, 0x100c9c8, 0x344b1e0)
	/go/src/github.com/gardener/gardener-extension-provider-alicloud/vendor/k8s.io/apimachinery/pkg/util/wait/wait.go:399 +0x55
k8s.io/client-go/util/retry.OnError(0x989680, 0x4014000000000000, 0x3fb999999999999a, 0x4, 0x0, 0x380b580, 0xc0009e3830, 0x32cfd00, 0x344b228)
	/go/src/github.com/gardener/gardener-extension-provider-alicloud/vendor/k8s.io/client-go/util/retry/util.go:50 +0xa6
k8s.io/client-go/util/retry.RetryOnConflict(...)
	/go/src/github.com/gardener/gardener-extension-provider-alicloud/vendor/k8s.io/client-go/util/retry/util.go:104
github.com/gardener/gardener/pkg/controllerutils.tryPatchFinalizers(0x39eca80, 0xc000da2c00, 0x0, 0x0, 0x2eb30ed8, 0xc0008273b0, 0x3a2ad40, 0xc00000d0e0, 0x380c800, 0x372ec10, ...)
	/go/src/github.com/gardener/gardener-extension-provider-alicloud/vendor/github.com/gardener/gardener/pkg/controllerutils/finalizers.go:60 +0x168
github.com/gardener/gardener/pkg/controllerutils.EnsureFinalizer(...)
	/go/src/github.com/gardener/gardener-extension-provider-alicloud/vendor/github.com/gardener/gardener/pkg/controllerutils/finalizers.go:49
github.com/gardener/gardener/extensions/pkg/controller/infrastructure.(*reconciler).reconcile(0xc000928910, 0x39eca80, 0xc000da2c00, 0x39f8560, 0xc000e10390, 0xc00000d0e0, 0xc00093b9e0, 0x36f92d7, 0x6, 0x0, ...)
	/go/src/github.com/gardener/gardener-extension-provider-alicloud/vendor/github.com/gardener/gardener/extensions/pkg/controller/infrastructure/reconciler.go:127 +0x11f
github.com/gardener/gardener/extensions/pkg/controller/infrastructure.(*reconciler).Reconcile(0xc000928910, 0x39eca80, 0xc000da2c00, 0xc000397b20, 0x1a, 0xc000f7d920, 0x9, 0x3a2ad40, 0xc00000cd20, 0x0, ...)
	/go/src/github.com/gardener/gardener-extension-provider-alicloud/vendor/github.com/gardener/gardener/extensions/pkg/controller/infrastructure/reconciler.go:121 +0x6c8
github.com/gardener/gardener/extensions/pkg/controller.(*operationAnnotationWrapper).Reconcile(0xc0009a6240, 0x39eca80, 0xc000da2c00, 0xc000397b20, 0x1a, 0xc000f7d920, 0x9, 0xc000da2c00, 0x100c29f, 0xc00003c000, ...)
	/go/src/github.com/gardener/gardener-extension-provider-alicloud/vendor/github.com/gardener/gardener/extensions/pkg/controller/reconciler.go:75 +0x259
sigs.k8s.io/controller-runtime/pkg/internal/controller.(*Controller).reconcileHandler(0xc0009977c0, 0x39ec9c0, 0xc000a3c440, 0x32cfde0, 0xc000d99cc0)
	/go/src/github.com/gardener/gardener-extension-provider-alicloud/vendor/sigs.k8s.io/controller-runtime/pkg/internal/controller/controller.go:263 +0x317
sigs.k8s.io/controller-runtime/pkg/internal/controller.(*Controller).processNextWorkItem(0xc0009977c0, 0x39ec9c0, 0xc000a3c440, 0x0)
	/go/src/github.com/gardener/gardener-extension-provider-alicloud/vendor/sigs.k8s.io/controller-runtime/pkg/internal/controller/controller.go:235 +0x205
sigs.k8s.io/controller-runtime/pkg/internal/controller.(*Controller).Start.func1.1(0x39ec9c0, 0xc000a3c440)
	/go/src/github.com/gardener/gardener-extension-provider-alicloud/vendor/sigs.k8s.io/controller-runtime/pkg/internal/controller/controller.go:198 +0x4a
k8s.io/apimachinery/pkg/util/wait.JitterUntilWithContext.func1()
	/go/src/github.com/gardener/gardener-extension-provider-alicloud/vendor/k8s.io/apimachinery/pkg/util/wait/wait.go:185 +0x37
k8s.io/apimachinery/pkg/util/wait.BackoffUntil.func1(0xc000bb6f50)
	/go/src/github.com/gardener/gardener-extension-provider-alicloud/vendor/k8s.io/apimachinery/pkg/util/wait/wait.go:155 +0x5f
k8s.io/apimachinery/pkg/util/wait.BackoffUntil(0xc0009e3f50, 0x398e960, 0xc000744060, 0xc000a3c401, 0xc000eb4cc0)
	/go/src/github.com/gardener/gardener-extension-provider-alicloud/vendor/k8s.io/apimachinery/pkg/util/wait/wait.go:156 +0xad
k8s.io/apimachinery/pkg/util/wait.JitterUntil(0xc000bb6f50, 0x3b9aca00, 0x0, 0x1, 0xc000eb4cc0)
	/go/src/github.com/gardener/gardener-extension-provider-alicloud/vendor/k8s.io/apimachinery/pkg/util/wait/wait.go:133 +0x98
k8s.io/apimachinery/pkg/util/wait.JitterUntilWithContext(0x39ec9c0, 0xc000a3c440, 0xc000a94bd0, 0x3b9aca00, 0x0, 0x1)
	/go/src/github.com/gardener/gardener-extension-provider-alicloud/vendor/k8s.io/apimachinery/pkg/util/wait/wait.go:185 +0xa6
k8s.io/apimachinery/pkg/util/wait.UntilWithContext(0x39ec9c0, 0xc000a3c440, 0xc000a94bd0, 0x3b9aca00)
	/go/src/github.com/gardener/gardener-extension-provider-alicloud/vendor/k8s.io/apimachinery/pkg/util/wait/wait.go:99 +0x57
created by sigs.k8s.io/controller-runtime/pkg/internal/controller.(*Controller).Start.func1
	/go/src/github.com/gardener/gardener-extension-provider-alicloud/vendor/sigs.k8s.io/controller-runtime/pkg/internal/controller/controller.go:195 +0x4e7
panic: runtime error: invalid memory address or nil pointer dereference [recovered]
	panic: runtime error: invalid memory address or nil pointer dereference [recovered]
	panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x18 pc=0x2e08bea]

goroutine 824 [running]:
k8s.io/apimachinery/pkg/util/runtime.HandleCrash(0x0, 0x0, 0x0)
	/go/src/github.com/gardener/gardener-extension-provider-alicloud/vendor/k8s.io/apimachinery/pkg/util/runtime/runtime.go:55 +0x10c
panic(0x321e8e0, 0x55259f0)
	/usr/local/go/src/runtime/panic.go:969 +0x1b9
k8s.io/apimachinery/pkg/util/runtime.HandleCrash(0x0, 0x0, 0x0)
	/go/src/github.com/gardener/gardener-extension-provider-alicloud/vendor/k8s.io/apimachinery/pkg/util/runtime/runtime.go:55 +0x10c
panic(0x321e8e0, 0x55259f0)
	/usr/local/go/src/runtime/panic.go:969 +0x1b9
github.com/gardener/gardener/pkg/controllerutils.tryPatchFinalizers.func1(0x10786bc, 0x552b1e0)
	/go/src/github.com/gardener/gardener-extension-provider-alicloud/vendor/github.com/gardener/gardener/pkg/controllerutils/finalizers.go:66 +0xea
k8s.io/client-go/util/retry.OnError.func1(0x0, 0x0, 0x0)
	/go/src/github.com/gardener/gardener-extension-provider-alicloud/vendor/k8s.io/client-go/util/retry/util.go:51 +0x3c
k8s.io/apimachinery/pkg/util/wait.runConditionWithCrashProtection(0xc0009e3790, 0x31fa000, 0x0, 0x0)
	/go/src/github.com/gardener/gardener-extension-provider-alicloud/vendor/k8s.io/apimachinery/pkg/util/wait/wait.go:211 +0x69
k8s.io/apimachinery/pkg/util/wait.ExponentialBackoff(0x989680, 0x4014000000000000, 0x3fb999999999999a, 0x4, 0x0, 0xc0009e3790, 0x100c9c8, 0x344b1e0)
	/go/src/github.com/gardener/gardener-extension-provider-alicloud/vendor/k8s.io/apimachinery/pkg/util/wait/wait.go:399 +0x55
k8s.io/client-go/util/retry.OnError(0x989680, 0x4014000000000000, 0x3fb999999999999a, 0x4, 0x0, 0x380b580, 0xc0009e3830, 0x32cfd00, 0x344b228)
	/go/src/github.com/gardener/gardener-extension-provider-alicloud/vendor/k8s.io/client-go/util/retry/util.go:50 +0xa6
k8s.io/client-go/util/retry.RetryOnConflict(...)
	/go/src/github.com/gardener/gardener-extension-provider-alicloud/vendor/k8s.io/client-go/util/retry/util.go:104
github.com/gardener/gardener/pkg/controllerutils.tryPatchFinalizers(0x39eca80, 0xc000da2c00, 0x0, 0x0, 0x2eb30ed8, 0xc0008273b0, 0x3a2ad40, 0xc00000d0e0, 0x380c800, 0x372ec10, ...)
	/go/src/github.com/gardener/gardener-extension-provider-alicloud/vendor/github.com/gardener/gardener/pkg/controllerutils/finalizers.go:60 +0x168
github.com/gardener/gardener/pkg/controllerutils.EnsureFinalizer(...)
	/go/src/github.com/gardener/gardener-extension-provider-alicloud/vendor/github.com/gardener/gardener/pkg/controllerutils/finalizers.go:49
github.com/gardener/gardener/extensions/pkg/controller/infrastructure.(*reconciler).reconcile(0xc000928910, 0x39eca80, 0xc000da2c00, 0x39f8560, 0xc000e10390, 0xc00000d0e0, 0xc00093b9e0, 0x36f92d7, 0x6, 0x0, ...)
	/go/src/github.com/gardener/gardener-extension-provider-alicloud/vendor/github.com/gardener/gardener/extensions/pkg/controller/infrastructure/reconciler.go:127 +0x11f
github.com/gardener/gardener/extensions/pkg/controller/infrastructure.(*reconciler).Reconcile(0xc000928910, 0x39eca80, 0xc000da2c00, 0xc000397b20, 0x1a, 0xc000f7d920, 0x9, 0x3a2ad40, 0xc00000cd20, 0x0, ...)
	/go/src/github.com/gardener/gardener-extension-provider-alicloud/vendor/github.com/gardener/gardener/extensions/pkg/controller/infrastructure/reconciler.go:121 +0x6c8
github.com/gardener/gardener/extensions/pkg/controller.(*operationAnnotationWrapper).Reconcile(0xc0009a6240, 0x39eca80, 0xc000da2c00, 0xc000397b20, 0x1a, 0xc000f7d920, 0x9, 0xc000da2c00, 0x100c29f, 0xc00003c000, ...)
	/go/src/github.com/gardener/gardener-extension-provider-alicloud/vendor/github.com/gardener/gardener/extensions/pkg/controller/reconciler.go:75 +0x259
sigs.k8s.io/controller-runtime/pkg/internal/controller.(*Controller).reconcileHandler(0xc0009977c0, 0x39ec9c0, 0xc000a3c440, 0x32cfde0, 0xc000d99cc0)
	/go/src/github.com/gardener/gardener-extension-provider-alicloud/vendor/sigs.k8s.io/controller-runtime/pkg/internal/controller/controller.go:263 +0x317
sigs.k8s.io/controller-runtime/pkg/internal/controller.(*Controller).processNextWorkItem(0xc0009977c0, 0x39ec9c0, 0xc000a3c440, 0x0)
	/go/src/github.com/gardener/gardener-extension-provider-alicloud/vendor/sigs.k8s.io/controller-runtime/pkg/internal/controller/controller.go:235 +0x205
sigs.k8s.io/controller-runtime/pkg/internal/controller.(*Controller).Start.func1.1(0x39ec9c0, 0xc000a3c440)
	/go/src/github.com/gardener/gardener-extension-provider-alicloud/vendor/sigs.k8s.io/controller-runtime/pkg/internal/controller/controller.go:198 +0x4a
k8s.io/apimachinery/pkg/util/wait.JitterUntilWithContext.func1()
	/go/src/github.com/gardener/gardener-extension-provider-alicloud/vendor/k8s.io/apimachinery/pkg/util/wait/wait.go:185 +0x37
k8s.io/apimachinery/pkg/util/wait.BackoffUntil.func1(0xc000bb6f50)
	/go/src/github.com/gardener/gardener-extension-provider-alicloud/vendor/k8s.io/apimachinery/pkg/util/wait/wait.go:155 +0x5f
k8s.io/apimachinery/pkg/util/wait.BackoffUntil(0xc0009e3f50, 0x398e960, 0xc000744060, 0xc000a3c401, 0xc000eb4cc0)
	/go/src/github.com/gardener/gardener-extension-provider-alicloud/vendor/k8s.io/apimachinery/pkg/util/wait/wait.go:156 +0xad
k8s.io/apimachinery/pkg/util/wait.JitterUntil(0xc000bb6f50, 0x3b9aca00, 0x0, 0x1, 0xc000eb4cc0)
	/go/src/github.com/gardener/gardener-extension-provider-alicloud/vendor/k8s.io/apimachinery/pkg/util/wait/wait.go:133 +0x98
k8s.io/apimachinery/pkg/util/wait.JitterUntilWithContext(0x39ec9c0, 0xc000a3c440, 0xc000a94bd0, 0x3b9aca00, 0x0, 0x1)
	/go/src/github.com/gardener/gardener-extension-provider-alicloud/vendor/k8s.io/apimachinery/pkg/util/wait/wait.go:185 +0xa6
k8s.io/apimachinery/pkg/util/wait.UntilWithContext(0x39ec9c0, 0xc000a3c440, 0xc000a94bd0, 0x3b9aca00)
	/go/src/github.com/gardener/gardener-extension-provider-alicloud/vendor/k8s.io/apimachinery/pkg/util/wait/wait.go:99 +0x57
created by sigs.k8s.io/controller-runtime/pkg/internal/controller.(*Controller).Start.func1
	/go/src/github.com/gardener/gardener-extension-provider-alicloud/vendor/sigs.k8s.io/controller-runtime/pkg/internal/controller/controller.go:195 +0x4e7
exit status 2
make: *** [start] Error 1
```

**What you expected to happen**:

**How to reproduce it (as minimally and precisely as possible)**:

1. Create Infrastructure

2. Observe the above panic

**Anything else we need to know?**:

**Environment**:

- Gardener version (if relevant):
- Extension version:
- Kubernetes version (use `kubectl version`):
- Cloud provider or hardware configuration:
- Others:

--- a/go.mod
+++ b/go.mod
@@ -11,7 +11,7 @@ require (
 	github.com/coreos/go-systemd/v22 v22.1.0
 	github.com/frankban/quicktest v1.9.0 // indirect
 	github.com/gardener/etcd-druid v0.3.0
-	github.com/gardener/gardener v1.18.1
+	github.com/gardener/gardener v1.18.2-0.20210318073350-b8027a3499c0
 	github.com/gardener/machine-controller-manager v0.36.0
 	github.com/go-logr/logr v0.3.0
 	github.com/gobuffalo/packr/v2 v2.8.1

